### PR TITLE
[Behat] Drastically reduced memory usage

### PR DESCRIPTION
**Pull request by pamil**
Based on #3627.

Eg. `checkout` suite dropped form 1.6GB to 250MB, now we can run whole Behat in one command! :)

### CODE REVIEW

File: app/TestKernel.php [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+            $serviceObject = new \ReflectionObject($service);
+            foreach ($serviceObject->getProperties() as $prop) {
+                $prop->setAccessible(true);
+                if ($prop->isStatic()) {
+                    continue;
+                }
+
+                try {
+                    $prop->setValue($service, null);
+                } catch (\ReflectionException $e) {
```
Comment (PWalkow) [inline app/TestKernel.php]: does it mean that sometimes not everything is being cleared?

File: behat.yml.dist [hunk 2]
```diff
@@ -400,16 +400,18 @@ default:
...
-        Behat\Symfony2Extension: ~
+        Behat\Symfony2Extension:
+            kernel:
+                class: TestKernel
+                path: app/TestKernel.php
 
     gherkin:
         filters:
-            tags: "~@javascript"
+            tags: "~@javascript && ~@contact"
```
Comment (PWalkow) [inline behat.yml.dist]: why this contact is so bad?
isn't easier to fix the contact suite?

File: bin/sylius-cs-fixer [hunk 3]
```diff
@@ -1,4 +0,0 @@
-#!/bin/bash
-
-php-cs-fixer fix . --config-file .php_cs
```
Comment (PWalkow) [inline bin/sylius-cs-fixer]: what happened to php-cs-fixer? does this change with behat really involves php-cs-fixing? if not maybe another PR would be better place for discussion?

File: behat.yml.dist (hunk 2 above)
Comment (pamil) [inline behat.yml.dist]: Contact is currently refactored in #3164.

File: app/TestKernel.php (hunk 1 above)
Comment (pamil) [inline app/TestKernel.php]: Don't know exactly, its slightly modified version of snippet from symfony/symfony#11236.

File: bin/sylius-cs-fixer (hunk 3 above)
Comment (pamil) [inline bin/sylius-cs-fixer]: :+1:, reverted

**Comment by pjedrzejewski** [comment]
Boom! This is just insane. Thanks Kamil!

**Pull request merged**